How-To-SSH-Tunnels.md

```diff
@@ -0,0 +1,109 @@
+# Using SSH tunnels to graph a remote Unix server
+
+Howto created by `fmangeant` at [Cacti
+Forum](http://forums.cacti.net/viewtopic.php?t=24960)
+
+This HowTo will explain how to use SSH tunnels to graph a Unix server that
+isn't directly reachable by your Cacti server :
+
+![Overview](images/ht_ssh_tunnel.png)
+
+In this example, the Cacti server can reach the Gateway, which can reach the
+target server.
+
+## Configuring the SSH tunnel
+
+On Gateway, create a "cactiuser" user :
+
+```console
+# useradd -d /home/cactiuser -m cactiuser
+```
+
+Then you have to generate SSH keys (without passphrase) :
+
+```console
+# su - cactiuser
+$ ssh-keygen -t rsa
+Generating public/private rsa key pair.
+Enter file in which to save the key (/home/cactiuser/.ssh/id_rsa):
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Your identification has been saved in /home/cactiuser/.ssh/id_rsa.
+Your public key has been saved in /home/cactiuser/.ssh/id_rsa.pub.
+The key fingerprint is:
+40:f6:91:a1:2d:d1:46:d4:76:e3:d8:c6:3f:c2:cf:f2 cactiuser@gateway
+```console
+
+The public key of the "cactiuser" user has then to be authorized as an input key :
+
+```console
+$ cd $HOME/.ssh
+$ cp -p id_rsa.pub authorized_keys
+```
+
+Finally, create the SSH tunnel :
+
+```console
+# su - cactiuser -c "ssh -f -N -g -L 192.168.0.2:10000:192.168.1.2:161 cactiuser@localhost"
+```
+
+This SSH tunnel will forward all packets sent to 192.168.0.2 on TCP port 10000,
+to 192.168.1.2 on port TCP 161.
+
+The options that are used are the following :
+
+```markdown
```

Comment:
should be shell or console... actually should probably be table...but I'll leave that up to you for now.